compiler/src/typed/oprint.re

```diff
@@ -631,22 +636,44 @@ and print_out_sig_item = ppf =>
       out_module_type^,
       mty,
     )
-  | Osig_type(td, rs) =>
-    print_out_type_decl(
-      switch (rs) {
-      | Orec_not => "type nonrec"
-      | Orec_first => "type"
-      | Orec_next => "and"
-      },
-      ppf,
-      td,
-    )
+  | Osig_type(td, _rs) => {
+      // TODO: Do we want/need these?
```

Comment:
I think we'll need this for recursive data types and stuff. I might want to patch this while whipping graindoc into shape.

I think I prefer to keep this as commented code and this TODO instead of opening an issue (since it really only affects the output of graindoc right now.